CSS/style.css

```diff
@@ -9,24 +9,90 @@ body{
     background: #ACC8E5;
     color: #112A46;
     font-family: 'Source Sans Pro', sans-serif;
-font-family: 'Work Sans', sans-serif;
+    font-family: 'Work Sans', sans-serif;
 }
 
+
+/* Submit button section  */
+.container{
+    overflow: hidden;
+    
+}
+
+.tab .tab-1 {
+    display: flex;
+}
+.tab-2{
+   margin-top: 1rem;
+}
+.tab .tab-2 {
+    display: flex;
+    margin-bottom: 10px;
+    flex-direction: column;
+}
+tr{
+    transition:all .25s ease-in-out
+}
+tr:hover{
+    background-color:#EEE;
+    cursor: pointer
+}
+
+.button-sec{
+    display: flex;
+    margin-left: 200px;
+}
+
+.button-sec{
+    display: inline-block;
+    margin-bottom: 1rem;
+}
+
+#show{
+    display: flex;
+    margin-bottom: 10px;
+}
+
+.edit-delete{
+    display: inline-block;
+    margin-left: 200px;
+    margin-top: 1rem;
+}
+
+#table{
+    display: inline-block;
+}
+
+
+
 section{
     margin: 25px;
     padding: 20px;
 }
 
+/* .job-1{
+    display: flex;
+} */
```

Comment:
Blocks of commented out code can be confusing: Maybe add a message to explain if/when it would be uncommented.